kolibri/plugins/management/assets/src/app.js

```diff
@@ -47,7 +48,7 @@ class ManagementModule extends KolibriModule {
     });
 
     router.start(rootvue, 'rootvue');
-    kolibri.coreActions.currentLoggedInUser(store, kolibri);
+    coreActions.currentLoggedInUser(store, kolibri);
```

Comment:
Ah, I see. So we would explicitly require a module like `core-actions` rather than referencing it off of the `kolibri` object. What was the rationale for preferring that pattern, vs referencing such things off of a facade-like object such as `kolibri`?